### PR TITLE
docs: set isChangeset import to validated-changeset

### PR DESCRIPTION
**Pull request by oliverlj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -888,7 +888,7 @@ changeset.save()
 If you're uncertain whether or not you're dealing with a `Changeset`, you can use the `isChangeset` util.
 
 ```js
-import isChangeset from 'ember-changeset/utils/is-changeset';
+import isChangeset from 'validated-changeset';
```
Comment (snewcomer) [inline README.md]: `import { isChangeset } ...`

**Comment by oliverlj** [comment]
sorry to come later but in typescript with

`import { isChangeset } from 'validated-changeset';`
I have an error : 
`Module '"../../../../../node_modules/validated-changeset/dist/types"' has no exported member 'isChangeset'. Did you mean 'IChangeset'?ts(2724)`

No problem with `import isChangeset from 'validated-changeset';`

**Comment by snewcomer** [comment]
@oliverlj Interesting.  Looks like we are exporting here....But that is a type error.  You could force the type for now.  I'll look into this today!

https://github.com/validated-changeset/validated-changeset/blob/08a4d514eff4df7bb57a2f5a2087f827fea3576f/src/index.ts#L37

**Comment by snewcomer** [comment]
I'm wondering if this is potentially the problem...

https://github.com/validated-changeset/validated-changeset/pull/16

**Pull request merged**